src/styles/index.scss

```diff
@@ -1,3 +1,239 @@
 body {
   margin: 0;
 }
+
+@font-face {
+  font-family: Roboto;
+  font-weight: 400;
+  src: url(../Roboto/Roboto-Regular.ttf);
+}
+
+@font-face {
+  font-family: Roboto;
+  font-weight: 500;
+  src: url(../Roboto/Roboto-Medium.ttf);
+}
+
+@font-face {
+  font-family: Roboto;
+  font-weight: 700;
+  src: url(../Roboto/Roboto-Bold.ttf);
+}
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50;
+  height: 60px;
+  box-shadow: 0 2 4 0 rgba(0, 0, 0, 0.05);
+
+  &__img {
+    width: 40px;
+    height: 40px;
+  }
+
+  &__container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    gap: 20px;
+  }
+
+  &__items {
+    font-family: Roboto, Arial, sans-serif;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 14px;
+    color: rgba(6, 11, 53, 1);
```

Comment:
[checklist](https://github.com/mate-academy/layout_catalog?tab=readme-ov-file#checklist): repaeted sizes and special colors are put to variables